docs/extend/extend-apiml/api-mediation-security.md

```diff
@@ -172,14 +168,14 @@ The `/login` endpoint authenticates mainframe user credentials and returns an au
   * Basic access authentication
```

Comment:
This is a general comment on documenting the ZAAS end-points in MD, while they should be documented with proper API doc in SWAGGER. Here we do it loosely, for example not all HTTP response codes are mentioned and not all possibilities are describerd. It is still OK to have it here, but may be we should state clearly that this is not to substitute the API documentation and that it can be found at <URI>